.github/workflows/tag-and-release.yml

```diff
@@ -0,0 +1,112 @@
+# This creates release on pushing with tag
+
+name: tag-and-release
+
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+defaults:
+  run:
+    working-directory: cli
+
+jobs:
+ 
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+            os: [ubuntu-latest, windows-2019, macos-latest ]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Get the version
+      id: get_version
+      run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+      shell: bash
+    - name: Create Release
```

Comment:
should not create the release before builds have completed imo.

Reply:
@fahad-israr you seem to still be creating the release before the build completed ?

Reply:
actually - I guess it is tricky doing this afterwards thus lets just live with it as its just a draft which we can delete if build fails for soem reason.